### ISSUE TITLE
Integrate with a cloud service to store data.

### ISSUE THREAD

**Issue by dhruv2601** [dhruv2601/Business-Card-Scanner#4]
The application scans Business Cards and extracts the details from it, and store them locally.
To enhance the application, it would be appreciable to add cloud services to it, so that user can access the data(Images and text) even after deleting the application.
Please integrate the application with those cloud services which comes at a lower cost, as the application is free and thus cloud costing should be minimal.

Feel free to contribute, continuous support will be given for understanding the code!

**Comment by akshatnitd** [comment]
@dhruv2601 It can be done using Firebase Storage, which gives quite decent storage under the free tier. It can also be done via AWS S3.

**Comment by dhruv2601** [comment]
@akshatnitd Yes! I will prefer Firebase myself. So are you up for working on it?

**Comment by akshatnitd** [comment]
@dhruv2601 No, you can go ahead with it. I am not that experienced with Android.
